CMakeLists.txt

```diff
@@ -21,6 +21,17 @@ endif()
 message(STATUS "Building in ${CMAKE_BUILD_TYPE} mode")
 message(STATUS "Building with ${CMAKE_C_COMPILER_ID} ${CMAKE_C_COMPILER_VERSION} on ${CMAKE_SYSTEM}")
 
+function(IsMSVC var)
+    if (MSVC OR (CMAKE_C_COMPILER_ID STREQUAL "Clang" AND CMAKE_C_SIMULATE_ID STREQUAL "MSVC"))
+        set(${var} 1 PARENT_SCOPE)
+    else()
+        set(${var} 0 PARENT_SCOPE)
+    endif()
+endfunction()
+
+set(is_msvc 0)
+IsMSVC(is_msvc)
```

Comment:
Just inline the var declaration instead of calling that parent scope-manipulating function? You're only using it once; less indirection is better, _ceteris paribus_.